Icon refresh

omanikhi opened the pull request.

omanikhi committed:
Overrode the action icons with the refreshed iconset

omanikhi committed:
Added the rest of the overridable actions but the ones with no corresponding icons are commented out

omanikhi committed:
Hid the View, Edit Contents and Use the URL actions

omanikhi committed:
The placeholder images on the preview page are now overridden with a SVG placeholder

takesson commented on branding/style/branding.css:
```diff
@@ -233,6 +236,9 @@
 }
 
 /* edit contents */
+#activities #btn_editfile {
```
Hmm, I would like to hide them on "details / open" page but keep them in "edit" page. 

Can we distinguish on "column-activities"?

omanikhi replied:
Sure we can. Done.

takesson replied:
Can you make the same change on use_url and open?

omanikhi replied:
Ah sorry missed the "them". Done.

omanikhi committed:
The btn_editfile, btn_open and btn_useurl are no longer hidden on the edit page

omanikhi committed:
Added unlock and list to the branding.css file

omanikhi committed:
Added some new icons from the latest delivery and replaced the rest with the updated versions

omanikhi committed:
Addressed the requested changes

 - Insert in Editor : Samma som Open in Editor
 - View in Repos: Använd View.svg (vi kommer ju ta bort den ena View-operationen)
 - Edit in Repos / Edit contents: edit_in_repos_and_contents.svg
 - Close Review: Ta samma som Open Review. Erik, bestämde vi inte att göra den utan rosa?
 - Revise graphic: Ajdå, den har vi missat. Ta Graphics_baseline.svg så får vi fundera på om det duger (dessa operationer är relaterade).
 - Export Translations: Skall tas bort, görs numera på Translation Prj sidan.
 - Batch Pretranslate: Skall tas bort
 - Pretranslate: Pretranslate_Report_pink.svg (sätt bättre namn)
 - Pretranslate Report kommer tas bort i CMS 5.0. Kommer eventuellt att återuppstå i CMS 5.x så vi lägger med  Pretranslate_Report_black.svg namngiven för Pretranslate Report.

omanikhi committed:
Added some padding to align the actions icons with the panels

omanikhi committed:
Copied and used the prepare_release.svg as validate_release.svg

omanikhi committed:
View in Repos was renamed to View file using a CSS hack

omanikhi committed:
The Copy historical icon was added using the same icon as copy.svg

omanikhi committed:
Added two new icons that were missing before
